.github/workflows/action-image-scan-deploy-dashing.yaml

```
@@ -0,0 +1,27 @@
+name: "docker-deploy-dashing"
+
+on: 
+  push:
+    branches:
+      - master
+      - main
+
+env:
+  ROS_VERSION: 2
+  UBUNTU_DISTRO: bionic
+  ROS_DISTRO: dashing
+
+jobs:
+  dockerhub-deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
```

what is the difference between this and `actions/checkout@v2`?

---

Good point. It is usually recommended to specify the version. I suggest changing it:
```suggestion
    - uses: actions/checkout@v2
```